include/arpa/inet.h

```diff
@@ -13,13 +13,19 @@ uint16_t htons(uint16_t);
 uint32_t ntohl(uint32_t);
 uint16_t ntohs(uint16_t);
 
-in_addr_t inet_addr (const char *);
-in_addr_t inet_network (const char *);
-char *inet_ntoa (struct in_addr);
-int inet_pton (int, const char *__restrict, void *__restrict);
-const char *inet_ntop (int, const void *__restrict, char *__restrict, socklen_t);
+in_addr_t inet_addr (const char *p : itype(_Nt_array_ptr<const char>));
+in_addr_t inet_network (const char *p : itype(_Nt_array_ptr<const char>));
+char *inet_ntoa (struct in_addr in) : count(15) itype(_Nt_array_ptr<char>);
```

Comment:
Why are the bounds `count(15)` here?

Reply:
This is commented above the implementation of `inet_ntoa` in ` src/network/inet_ntoa.c`.